Examples/Swift/ChangeWorldviewBoundariesExample.swift

```diff
@@ -0,0 +1,76 @@
+import UIKit
+import Mapbox
+
+@objc(ChangeWorldviewBoundariesExample_Swift)
+
+class ChangeWorldviewBoundariesExample: UIViewController, MGLMapViewDelegate {
+
+    let mapView = MGLMapView()
+    let source = MGLVectorTileSource(identifier: "admin-bounds", configurationURL: NSURL(string: "mapbox://mapbox.mapbox-streets-v8")! as URL)
+    var adminLayers = [String]()
+    var layer: MGLLineStyleLayer?
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        mapView.frame = CGRect(x: 0, y: 0, width: view.bounds.width, height: view.bounds.height)
```

Comment:
You can assign this to `self.view.frame` instead of using the entire `CGRect` constructor.